workers/Dockerfile.base

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:16.04
+
+RUN \
+  apt-get update -qq && \
+  apt-get install -y \
+                     lsb-release && \
+  echo "deb http://archive.ubuntu.com/ubuntu $(lsb_release -sc) multiverse\n" \
+      >> /etc/apt/sources.list.d/added_repos.list && \
+  echo "deb http://cran.cnr.berkeley.edu/bin/linux/ubuntu $(lsb_release -sc)/" \
+      >> /etc/apt/sources.list.d/added_repos.list && \
+  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9 && \
```

Comment:
Seems to be the recommended way to install R: see the `Secure APT` section of: https://cran.r-project.org/bin/linux/ubuntu/